stable/spring-cloud-data-flow/templates/skipper-config.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if not .Values.skipper.configMap }}
```

Comment:
`skipper-deployment.yaml` would need to be updated as with `server-deployment.yaml`

Reply:
Done